massa-execution-worker/src/execution.rs

```diff
@@ -426,9 +467,12 @@ impl ExecutionState {
                 })?;
                 *block_credits = block_credits.saturating_add(amount);
             }
-            Err(e) => warn!("Unable to slash rolls or deferred credits: {}", e),
+            Err(e) => {
+                warn!("Unable to slash rolls or deferred credits: {}", e);
+            }
         }
 
+        context.insert_executed_de(&de_idx);
```

Comment:
more generally, I understand that "de" is more compact but I vote for using the full word "denunciation" or at least "denunc" because reviewers/readers will struggle to understand what it is

Reply:
I've tried to rename most function names & struct field names with 'denunciation' instead of 'de'.